demo/index.html

```diff
@@ -126,21 +127,31 @@
         `;
       }
     }
-    customElements.define('menu-content', MenuContent);
-    const menus = [{
-      icon: html`
-        <ia-icon icon="audio"></ia-icon>
-      `,
-      title: 'Audio Menu',
-      menuDetails: '(100,000,000 tracks)',
-      actionButton: html`
+    let showPrimary = true;
+    const primaryActionButton = html`
+        <button
+        style='--iconFillColor: aquamarine;'
+          @click=${e => extraActionClickHandler(e)}
+        >
+          <ia-icon-volumes></ia-icon-volumes>
+        </button>
+      `;
+    const secondaryActionButton = html`
         <button
-          style='color: white;'
-          @click=${e => console.log('CLICK EXTRA ACTION', e)}
+          style='--iconFillColor: red;'
+          @click=${e => extraActionClickHandler(e)}
         >
           <ia-icon-volumes></ia-icon-volumes>
         </button>
+      `;
+customElements.define('menu-content', MenuContent);
```

Comment:
are we not going to fix the indentation here? this is one of the things i stumble upon when we do linting/prettier or even use code formatter plugins in our editors

Reply:
:heavy_plus_sign:

Reply:
thanks for catching!

Reply:
@dualcnhq this is a good catch, can you create a ticket to add these code formatters to our template repos?